readme.md

```diff
@@ -35,11 +35,11 @@ GitHub username.
 
 #### token
 
-Type: `string`  
```

Comment:
The two spaces at the end of the line?
Do you want the same space added at the end of the type: string above for the username argument?